eventually/tests/unittests/event/testmodel.py

```diff
@@ -0,0 +1,220 @@
+                        owner=custom_user,
+                        name='barcelona')
+            team.save()
+
+            event = Event(id=101,
+                          team=team,
+                          owner=custom_user,
+                          name='football')
+            event.save()
+
+            event = Event(id=102,
+                          team=team,
+                          owner=custom_user,
+                          name='football',
+                          description='test description',
+                          start_at=TEST_TIME,
+                          duration=datetime.timedelta(seconds=12000),
+                          longitude=-61.523438,
+                          latitude=-61.523438,
+                          budget=100,
+                          status=2)
+            event.save()
+
+    def test_event_to_dict(self):
+        """Method that tests `to_dict` method of certain Event instance."""
+
+        event = Event.objects.get(id=101)
+        expect_event_dict = {
+            'id': 101,
+            'team': 101,
+            'owner': 101,
+            'name': 'football',
+            'description': '',
+            'start_at': None,
+            'created_at': 1508044512,
+            'updated_at': 1508044512,
+            'duration': None,
+            'longitude': None,
+            'latitude': None,
+            'budget': None,
+            'status': 0,
+        }
+
+        actual_event_dict = event.to_dict()
+
+        self.assertDictEqual(actual_event_dict, expect_event_dict)
+
+    def test_event_all_parameters_to_dict(self):
+        """
+        Method that tests `to_dict` method of certain Event instance.
+
+        Function tests Event instance with all filled parameters.
+        """
+
+        event = Event.objects.get(id=102)
+        expected_event_dict = {
+            'id': 102,
+            'team': 101,
+            'owner': 101,
+            'name': 'football',
+            'description': 'test description',
+            'start_at': 1508044512,
+            'created_at': 1508044512,
+            'updated_at': 1508044512,
+            'duration': 12000,
+            'longitude': -61.523438,
+            'latitude': -61.523438,
+            'budget': 100,
+            'status': 2,
+        }
+
+        actual_event_dict = event.to_dict()
+
+        self.assertDictEqual(actual_event_dict, expected_event_dict)
+
+    def test_event_success_get_by_id(self):
+        """Method that tests succeeded `get_by_id` method of Event class object."""
+
+        actual_event = Event.get_by_id(101)
+        expected_event = Event.objects.get(id=101)
+
+        self.assertEqual(actual_event, expected_event)
+
+    def test_event_none_get_by_id(self):
+        """Method that tests unsucceeded `get_by_id` method of Event class object."""
+
+        actual_event = Event.get_by_id(123)
+
+        self.assertIsNone(actual_event)
+
+    def test_event_success_create(self):
```

Comment:
You have try/except for 2 excetptions (ValueError and Integrity Error)in in create method in your models.py. Raise them for full coverage.

Reply:
Do you mean the exceptions that rise when I try to save the object to db? If yes I test this case in the `test_event_none_create` method, but there is `ValueError` when I don't send Team and CustomUser objects as first and second parameters and I don't know is it the test case or incorrect models file logic.